.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug
+about: To report a bug found in the bot's working.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Description**
+
+<!-- A brief description of the question or issue, also include what you tried and what didn't work: -->
+
+**Screenshots**
+
+<!-- Please add a screenshot if applicable -->
+
+**Additional Context**  <!-- Optional -->  
+
+<!-- Add any other context about the problem here. -->
+
+Join Eddie's discord community [here](https://discord.com/invite/jZQs6Wu)
```

Comment:
I am not sure if we should include the link to the discord
(this is a project for the client and maybe we should not do this?)

*maybe we can get a second opinion on this from someone ?

Reply:
hmm interesting, I think this is ok for now, we can always remove later

Reply:
Okay - I'll leave it there for now 👍🏻

Reply:
:+1: